hack/tools/release/notes/print.go

```diff
@@ -79,7 +81,11 @@ func (p *releaseNotesPrinter) print(entries []notesEntry) {
 		fmt.Printf("🚨 This is a RELEASE CANDIDATE. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/%s/issues/new).\n", p.repo)
```

Comment:
This could either be a release candidate or beta release. You can either change this based on how the release tag string is formatted, or just include both `RELEASE CANDIDATE/BETA RELEASE` and have the user manually change this.